hack/codegen.sh

```diff
@@ -17,10 +17,16 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-${scriptdir}/generate-groups.sh \
-  deepcopy,client,lister,informer \
-  github.com/pingcap/tidb-operator/pkg/client \
-  github.com/pingcap/tidb-operator/pkg/apis \
-  "pingcap:v1alpha1"
+export GO111MODULE=on
+
+go mod vendor
```

Comment:
populate `vendor` directory

Reply:
Can we use `retool` for the installed binaries?